src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,14 +112,22 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
+           string caption;
+            if(!string.IsNullOrEmpty(this.Title)) {
+                caption = this.Title;
+            }
+            else {
+                caption = UtilsStrings.PromptForCredential_DefaultCaption;
+            }
```

Comment:
If you add `ValidateNotNullOrEmpty` attribute to `Title`, then this can be turned to
```
string caption = this.Title ?? UtilsStrings.PromptForCredential_DefaultCaption; 
```